### DISCUSSION TITLE
How to use kubernetes-client in .Net Framework ?

### DISCUSSION THREAD

**Discussion by losha228** [kubernetes-client/csharp#1165] [Q&A] [answered]
I am trying to find a kube client for .Net Framework,  my project uses .Net framework 4.6.2, I checked the nuget pkg dependency, it seems it doesn't support .Net framework, right ?
How can I leverage this project to support .Net framework ?  Thanks.

**Comment by tg123** [comment, marked as answer]
we do support netframework
see https://www.nuget.org/packages/KubernetesClient.Classic
